### PR TITLE
Feat/13 organization settings roles and permission page implement create role modal

### PR DESCRIPTION
**Pull request by JO-YE**
## Ticket/Issue number
#13 

## **Task**
- [x] Create a modal component
- [X] Add a title "Create Role"
- [X] Add an input field for the name with the placeholder "e.g: IT Staff"
- [X] Add an input field for the description with the placeholder "describe role"
- [X] Add a "Cancel" button
- [X] Add a "Create Role" button
- [X] Ensure the modal follows brand colors and design
- [X] Ensure the modal is responsive

## Screenshot of Create Role Modal Implementation

create_role_modal

Using-Shadcn-modal

[Figma Link](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=118-44881&m=dev)

### CODE REVIEW

File: app/components/ui/create-role-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  className = "",
+  onClose,
+}) => {
+  return (
+    <div
+      role="dialog"
+      aria-modal="true"
+      aria-labelledby="modal-title"
+      className={`m-auto flex min-h-[435px] max-w-[465px] flex-col rounded-md bg-zinc-50 px-6 py-6 shadow-lg ${className}`}
+    >
```
Comment (chumex412) [inline app/components/ui/create-role-modal.tsx:21]: Please use the ShadCN modal instead of this custom implementation.

**Review by chumex412 — DISMISSED** [review]

File: app/components/ui/create-role-modal.tsx (hunk 1 above)
Comment (JO-YE) [reply app/components/ui/create-role-modal.tsx:21]: Alright

Comment (JO-YE) [reply app/components/ui/create-role-modal.tsx:26]: Done.

Comment (chumex412) [reply app/components/ui/create-role-modal.tsx:26]: You say done but I don't see anything

**Comment by JO-YE** [comment]
this branch "JO-YE:feat/13-organization-settings-roles-and-permission-page-implement-create-role-modal" is updated with the changes

**Review by shalomtaiwo — APPROVED** [review]
good

**Comment by JO-YE** [comment]
Thank you @shalomtaiwo

**Pull request merged**